scripts/install_geospatial.sh

```diff
@@ -72,8 +76,6 @@ install2.r --error --skipmissing --skipinstalled -n "$NCPUS" \
     terra \
     tidync \
     tmap \
-    geoR \
-    geosphere \
```

Comment:
Removing the R packages will affect previous R version images, is that as intended?